ihatemoney/models.py

```diff
@@ -382,13 +394,27 @@ def pay_each(self):
         else:
             return 0
 
+    def is_loan(self):
+        return self.type == BillTypes.loan
```

Comment:
`== "loan"`

Reply:
I don't get why you don't want to use a variable instead of using a raw string :thinking:

Reply:
As I said elsewhere, it's to ease the logic. That's the more "pythonic" way of doing things. It's more idiomatic. It's less useless code floating around :-)

Reply:
But that means if you want to change `"loan"` by `"bill"` for example you will  have to update each `"loan"` occurrence.
Doesn't seem a bad practice to you ?
If it's more "pythonic" I can remove the class `BillTypes` and replace `BillTypes.loan` by `"loan"` and `BillType.settlement` by `"settlement"`.
It doesn't fit the way I'm used to code but if it makes the code more readable why not. 🤷‍♂

Reply:
If we were to change "loan" by "bill" for instance, then we would just have to find all occurrences and update them, really no big deal.

I believe it would make the code more readable. I can understand why this would be the way to do things in Java for instance, but in python, it's easier to do it this way I believe :-) Let's do it !